bot/package.json

```diff
@@ -1,34 +1,37 @@
 {
   "name": "airhornbot",
   "version": "1.0.0",
+  "description": "",
   "main": "index.js",
-  "author": "Discord",
+  "author": "Sky W",
   "license": "MIT",
-  "private": true,
+  "type": "module",
   "scripts": {
-    "register-commands": "node ./dist/register",
-    "bot": "node ./dist/bot",
-    "web": "node ./dist/web",
-    "build": "tsc --project ./tsconfig.json",
-    "lint": "eslint . --ext .ts,.tsx"
+    "build": "tsc --project tsconfig.json",
```

Comment:
The `--project tsconfig.json` but is the default.  A `tsc` should do fine here :)